### DISCUSSION TITLE
Synchronization becomes slow

### DISCUSSION THREAD

**Discussion by lilin-lab** [filecoin-project/lotus#6902] [Build and Sync]
lotus data was collected for 10 seconds using Pprof, found a lot of time consuming in loadTipSet
![image](https://user-images.githubusercontent.com/26270251/127255897-b18ecb8c-c609-4ce2-ab12-a4435de9667d.png)

**Comment by de-authority** [comment]
Which version do you use？　Is it possible that it is caused by hardware

**Reply by lilin-lab** [reply]
v1.10.1

**Reply by lilin-lab** [reply]
What kind of hardware is needed，thank you

**Comment by LexLuthr** [comment]
@johnli-helloworld You can read through the hardware requirements for lotus-miner in the below link.
https://lotu.sh/docs/storage-providers/hardware-requirements/
https://lotu.sh/
